css/main.css

```diff
@@ -0,0 +1,212 @@
+
+ body{
+  font-size: 100%;
+  background-image: url(../images/homepic.jpg);
+  background-size: cover;
+  font-style: oblique;
+}
+
+.navigation {
+ /* background-color:  #D3D3D3; */
+ overflow: hidden;
+}
+
+.navigation a {
+   text-align: right;
+   color: black;
+  text-align: center;
+  padding: 10px 16px;
+  text-decoration: none;
+  font-size: 17px;
+}
+
+.navigation ul{
+    float: right;
+}
+
+
+ .navigation a:hover {
+  color: black;
+}
+
+.navigation img{
+  float: left;
+  height: 60px;
+  width: 60px;
+  color: black;
+
+}
+
+.content{
+    text-align: center;
+    
+    
+}
+
+.greetings{
+  margin-top: 0.5px;
+  margin-left: 50px;
+
+}
+
+.greetings h1 {
+  margin-top: 0.5%;
+  padding-top: 10%;
+  padding-bottom: 50px;
+  align-items: center;
+  color: black;
+  
+  
+  
+}
+.greetings p{
+  color: burlywood;
+  margin-top: 0.5%;
+  padding-top: 10%;
+  padding-bottom: 50px;
+  align-items: center;
+}
+
+.greetings h1{
+  margin-left: 25%;
+}
+
+h1{
+   font-size: 350%;
+}
+
+p{
+   font-size: 150%;
+   font-family: 'Quicksand', sans-serif;
+}
+
+  
+}
+
+ .visible{
+  color: black;
+}
+
+.log-in-form{
+  text-align: center;
+  text-align: justify;
+  text-justify: auto;
+
+}
+
+* {
+    box-sizing: border-box;
+  }
+  
+  input[type=text], select, textarea {
+    width: 100%;
+    padding: 12px;
+    border: 1px solid #ccc;
+    border-radius: 4px;
+    resize: vertical;
+  }
+  
+  label {
+    padding: 12px 12px 12px 0;
+    display: inline-block;
+  }
+  
+  input[type=submit] {
+    background-color: #4CAF50;
+    color: white;
+    padding: 12px 20px;
+    border: none;
+    border-radius: 4px;
+    cursor: pointer;
+    float: right;
+  }
+  
+  input[type=submit]:hover {
+    background-color: #4CAF50
+  }
+  
+  .container {
+    border-radius: 5px;
+    background-color: #f2f2f2;
+    padding: 20px;
+  }
+  
+  .col-25 {
+    float: left;
+    width: 25%;
+    margin-top: 6px;
+  }
+  
+  .col-75 {
+    float: left;
+    width: 75%;
+    margin-top: 6px;
+  }
+  
+  /* Clear floats after the columns */
+  .row:after {
+    content: "";
+    display: table;
+    clear: both;
+  }
+  
+  /* Responsive layout - when the screen is less than 600px wide, make the two columns stack on top of each other instead of next to each other */
+  @media screen and (max-width: 600px) {
+    .col-25, .col-75, input[type=submit] {
+      width: 100%;
+      margin-top: 0;
+    }
+  }
+ 
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
```

Comment:
@veemish  this is supposed to be in your html file

Reply:
okay, I will have have to review and get it back to you, am not sure why the link is in the CSS file. That's what you mean I hope.@Benmaruchu, I will find out more.